_includes/head.html

```diff
@@ -8,7 +8,7 @@
 
   {% assign custom_url = site.url | append: site.baseurl %}
   {% assign full_base_url = custom_url | default: site.github.url %}
-  <link rel="stylesheet" href="{{ "/css/main.css" | prepend: full_base_url }}">
+  <link rel="stylesheet" href="{{ "/assets/css/main.css" | prepend: full_base_url }}">
```

Comment:
I'm advocating `assets/css/main.scss`, so the following is in context to that:
### pros
- this is a breaking change (to me, at least). I feel, its the right time to initiate the practice of having delegated folders. `css` for stylesheets, `js` for scripts, `fonts` for well.. fonts. 
- having this set right now will show users/developers that Jekyll 3.3 supports delegated folders. minima is not just a theme, its also a boilerplate / reference for future theme gems.

### cons
- having to create two folders to house one file, while preparing to override default settings.
- (or), in the case of existing sites, having to create `assets/` and move their `css/` to it, instead of simply renaming `css/ -> assets/`